### PR TITLE
[IMP] Documents: share section to be improved

### PR DESCRIPTION
**Pull request by Donapi**
taskid-[3898917](https://www.odoo.com/web#id=3898917&cids=1&model=project.task&view_type=form)

Applicable  in 17  and 17.1

### CODE REVIEW

**Review by toaa-odoo — APPROVED** [review]

File: content/applications/productivity/documents.rst [hunk 1]
```diff
@@ -184,19 +194,73 @@ To add a link to your documents dashboard, click on :guilabel:`Add a Link`, ente
...
+
+To allow the upload of documents by email, you can create an email alias. The domain name is set by
+default but can be modified by clicking on it. The documents sent to this email alias are uploaded
+using the chosen tags to the workspace.
+
+By default, the :guilabel:`Document Owner` is the person who uploads a file to a workspace, but you
+can select someone else. You can also set a :guilabel:`Contact`, usually an external person, such as
+a partner.
+
+You can check the :guilabel:`Create a new activity` box to automatically create an activity when a
```
Comment (auva-odoo) [inline content/applications/productivity/documents.rst:257]:
```suggestion
Enable :guilabel:`Create a new activity` to automatically create an activity when a
```
for the sake of simplifying the sentence

**Review by auva-odoo — CHANGES_REQUESTED** [review]
Hello @Donapi thank you for your work! I have a few comments and questions so let me know if you want to go through them together.
One general comment I also have regarding this whole doc page is that we could add the icons directly in the text instead of only describing them, now that it's possible https://www.odoo.com/documentation/master/contributing/documentation/rst_cheat_sheet.html#icons . I don't think this should be done in this PR, though, but it might be worth creating a task to do this (e.g., when we say "ellipsis icon", "plus icon", "scissors icon", etc.), possibly from 16.0? We can talk about it!

File: content/applications/productivity/documents.rst [hunk 2]
```diff
@@ -114,16 +116,20 @@ an :guilabel:`Owner` can be assigned. The related :guilabel:`Workspace` can be m
...
-     view the document and not modify it. I.e.: an existing supplier in your database is the contact
-     for their bill.
-   - The person who creates a document is, by default :guilabel:`Owner` of it and has complete
-     rights to it. It is possible to replace the owner of a document. I.e., an employee
-     must be owner of a document to be able to see it in "My Profile".
+   - The :guilabel:`Contact` is a person related to the document, who has only viewing access rights
+     to the document. I.e.: an existing supplier in your database.
+   - The person who creates a document is by default :guilabel:`Owner` and has full access rights to
+     it. It is possible to replace the owner of a document by selecting someone else from the
+     dropdown list of the :guilabel:`Owner` field.
```
[the remark this answers is not in the document]
Comment (Donapi) [reply content/applications/productivity/documents.rst:123]: Thank you for the suggestion, it makes sense to me, I will keep it in mind :)

File: content/applications/productivity/documents.rst [hunk 3]
```diff
@@ -158,7 +164,9 @@ Select the workspace where the file should be stored, click :guilabel:`Upload`,
 :guilabel:`Note`, if needed. Then, click :guilabel:`Request`. A placeholder for the missing document
 is created in the workspace.
 
-When your document is available, click the placeholder to upload it.
+When your document is available, click the placeholder to upload it in its stead. By default, you
+are set as the document :guilabel:`Owner`, and the requested person is set as the
+:guilabel:`Contact`.
```
[the remark this answers is not in the document]
Comment (Donapi) [reply content/applications/productivity/documents.rst:169]: The requested person can indeed upload the document; however, I double-checked, and they do not become the Contact person for the document.

[the remark this answers is not in the document]
Comment (Donapi) [reply content/applications/productivity/documents.rst:169]: I tried to simplify the sentence.

File: content/applications/productivity/documents.rst [hunk 4]
```diff
@@ -184,19 +194,73 @@ To add a link to your documents dashboard, click on :guilabel:`Add a Link`, ente
...
+
+Upload document
+^^^^^^^^^^^^^^^
+
+Select the :guilabel:`Download and Upload` option to enable users to upload their files to your
+workspace.
+
+To allow the upload of documents by email, you can create an email alias. The domain name is set by
+default but can be modified by clicking on it. The documents sent to this email alias are uploaded
+using the chosen tags to the workspace.
```
[the remark this answers is not in the document]
Comment (Donapi) [reply content/applications/productivity/documents.rst:251]: I absolutely agree. Updated accordingly.

File: content/applications/productivity/documents.rst [hunk 5]
```diff
@@ -114,16 +116,20 @@ an :guilabel:`Owner` can be assigned. The related :guilabel:`Workspace` can be m
...
-     rights to it. It is possible to replace the owner of a document. I.e., an employee
-     must be owner of a document to be able to see it in "My Profile".
+   - The :guilabel:`Contact` is a person related to the document, who has only viewing access rights
+     to the document. I.e.: an existing supplier in your database.
+   - The person who creates a document is by default :guilabel:`Owner` and has full access rights to
+     it. It is possible to replace the owner of a document by selecting someone else from the
+     dropdown list of the :guilabel:`Owner` field.
+
+.. tip::
+   An employee must be owner of a document to be able to see it in **My Profile**.
```
[the remark this answers is not in the document]
Comment (Donapi) [reply content/applications/productivity/documents.rst]: Feedback from FRGI: employees must be users to see a document.

**Review by auva-odoo — APPROVED** [review]
@Donapi I force-pushed 3 minor corrections, I hope that's okay :slightly_smiling_face: 
@robodoo r+

**Pull request closed without merging**